code/drasil-website/lib/Drasil/Website/About.hs

```diff
@@ -99,6 +100,10 @@ aboutParagraph8 paperGOOL = S "As described in the" +:+ namedRef paperGOOL (S "G
   \generation, allowing Drasil to more efficiently generate code in several languages, including Python, Java, C-Sharp, and C++."
 
 -- | Paragraph providing a link to Drasil papers and documents
-aboutParagraph9 :: Reference -> Sentence
-aboutParagraph9 papersWiki = S "A list of papers and documents written about Drasil can be found in the" +:+ namedRef papersWiki (S "Drasil Papers \
-  \and Documents") +:+ S "wiki page"
```

Comment:
The reuse of `S "wiki page"` (4 uses in Drasil total) makes me wonder if we want to formally encode it as a named concept (probably in documentation)....

Reply:
@smiths Do you know if we have an issue tracking this sort of thing? I could just make a quick <kbd>[newcomers](https://github.com/JacquesCarette/Drasil/labels/newcomers)</kbd> issue for this one change, but I feel like this is part of a potentially bigger issue (investigating what meaningful concepts are still manually created in examples, etc.)

Reply:
@samm82 I'm not aware of an issue for tracking new named concepts.  My guess is that we don't have this, but we might.  We are going to need more newcomer issues when the summer starts, so it is probably best to just create the newcomer issue that you mentioned.  If there are other related isses, we can add the traceability later.

Reply:
Created in #3698!